src/sentry/integrations/vsts/integration.py

```diff
@@ -42,10 +42,11 @@ def get_project_config(self):
         disabled = False
         try:
             projects = client.get_projects(self.model.metadata['domain_name'])
-        except ApiError:
+        except Exception:
```

Comment:
I can change it back. I was just thinking "what if the metadata doesn't have a domain_name" for whatever reason. Then the whole page fails to load. What might be better is to have the endpoint that calls this function do the error handling instead tbh. Because if you can't get to this page, you can't delete the app altogether. And I think that's an undesirable result

Reply:
Let me know what you think though.